### ISSUE TITLE
Save downloaded music for playing in an external music player 


### ISSUE THREAD

**Issue by VahidDopler** [Arturo254/InnerTune#141]
Hi
First I should thank you for making this great app with all its benefits . I have downloaded a sound so , I want to play it in another music player , I guess it is stored in another place , what should I do ?

**Comment by Arturo254** [comment]
No. The goal is to make downloaded songs weigh less than if you download them in mp3, opus, wav, etc. format.

It works like Spotify

**Comment by ungive** [comment]
Would also like to see this! I can't really use any music app that doesn't take control of my external DAC because on my specific phone model that specific DAC has crackling noises when used through Android's sound mixer.

Your app looks stunning btw, great job.
